blocks/product-details/product-details.js

```diff
@@ -132,13 +144,13 @@ export default async function decorate(block) {
 
     // Configuration – Button - Add to Cart
     UI.render(Button, {
-      children: labels.PDP?.Product?.AddToCart?.label,
+      children: labels.PDP?.Product?.AddToCart?.label || 'Add to Cart',
```

Comment:
```suggestion
      children: labels.PDP?.Product?.AddToCart?.label,
```
Since this is an existing label which should already be defined in the boilerplate content source, we don't need to add this fallback.